src/megachatapi_impl.cpp

```diff
@@ -2679,11 +2719,11 @@ void MegaChatApiImpl::getUserEmail(MegaChatHandle userhandle, MegaChatRequestLis
     waiter->notify();
 }
 
-void MegaChatApiImpl::loadUserAttributes(MegaChatHandle chatid, MegaHandleList userList, const char *authorizationToken, MegaChatRequestListener *listener)
+void MegaChatApiImpl::loadUserAttributes(MegaChatHandle chatid, MegaHandleList *userList, const char *authorizationToken, MegaChatRequestListener *listener)
 {
     MegaChatRequestPrivate *request = new MegaChatRequestPrivate(MegaChatRequest::TYPE_GET_PEER_ATTRIBUTES, listener);
     request->setChatHandle(chatid);
-    request->setMegaHandleList(userList.copy());
+    request->setMegaHandleList(userList->copy());
```

Comment:
You are not testing `userList` (it could be null). Apart from that, the `setMegaHandleList()` already does a copy, so it's a waste to pass a copy here.

Reply:
Fixed